app/assets/stylesheets/govuk_publishing_components/component_guide.scss

```diff
@@ -132,3 +136,33 @@ $border-color: #ccc;
     }
   }
 }
+
+// Preview Page Styling
+html {
+  background: $white;
+}
+
+.hide-header-and-footer {
+  // scss-lint:disable IdSelector
+  #global-header,
+  #global-header-bar,
+  #global-breadcrumb,
+  #footer {
+    display: none;
+  }
+}
+
+.component-guide-preview-page {
+  padding: ($gutter * 1.5) $gutter $gutter;
```

Comment:
Is there a reason for this element to have padding left and right? It's already wrapped, so it should never collide with the edge of the browser window.